python-checks/src/test/java/org/sonar/python/checks/tests/AssertAfterRaiseCheckTest.java

```diff
@@ -42,4 +43,20 @@ public void testWithAnotherLibraryUnittest() {
   public void testImportPytestAs() {
     PythonCheckVerifier.verify("src/test/resources/checks/tests/assertAfterRaiseImportPytestAs.py", new AssertAfterRaiseCheck());
   }
+
```

Comment:
Can you also add a small test asserting we don't provide quick fixes for single statement bodies?

Also, probably a silly edge case as well, but what if all the statements are on the same line?
```
import pytest
def test_smth():
  with pytest.raises(ZeroDivisionError): foo(); assert bar() == 42
```
Nobody writes code like this, but if anyone ever does, I think it'd be better to not provide any quick fixes than to possibly break the code.

Reply:
done